docs/source/reference/kubernetes/kubernetes-setup.rst

```diff
@@ -196,11 +200,223 @@ Use the following command to label a node:
 
     GPU labels are case-sensitive. Ensure that the GPU name is lowercase if you are using the ``skypilot.co/accelerator`` label.
 
+.. _kubernetes-setup-verify:
+
+Verifying setup
+---------------
+
+Once the cluster is deployed and you have placed your kubeconfig at ``~/.kube/config``, verify your setup by running :code:`sky check`:
+
+.. code-block:: bash
+
+    sky check kubernetes
+
+This should show ``Kubernetes: Enabled`` without any warnings.
+
+You can also check the GPUs available on your nodes by running:
+
+.. code-block:: console
+
+    $ sky show-gpus --cloud k8s
+    Kubernetes GPUs
+    GPU   REQUESTABLE_QTY_PER_NODE  TOTAL_GPUS  TOTAL_FREE_GPUS
+    L4    1, 2, 4                   12          12
+    H100  1, 2, 4, 8                16          16
+
+    Kubernetes per node GPU availability
+    NODE_NAME                  GPU_NAME  TOTAL_GPUS  FREE_GPUS
+    my-cluster-0               L4        4           4
+    my-cluster-1               L4        4           4
+    my-cluster-2               L4        2           2
+    my-cluster-3               L4        2           2
+    my-cluster-4               H100      8           8
+    my-cluster-5               H100      8           8
+
+.. _kubernetes-optional-steps:
+
+Optional setup
+--------------
+
+The following setup steps are optional and can be performed based on your specific requirements.
+
+
+.. _kubernetes-setup-volumes:
+
+Setup NFS and other volumes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`Kubernetes volumes <https://kubernetes.io/docs/concepts/storage/volumes/>`_ can be attached to your SkyPilot pods using the :ref:`pod_config <kubernetes-custom-pod-config>` field. This is useful for accessing shared storage such as NFS or local high-performance storage like NVMe drives.
+
+Volume mounting can be done directly in the task YAML on a per-task basis, or :ref:`globally for all tasks<kubernetes-custom-pod-config>` in :code:`~/.sky/config.yaml`.
+
+Examples:
```

Comment:
The nested tabs seem to have some weird caching of which inner tab is selected, and the clearing behavior is not intuitive. Maybe flatten inner tabs into each snippet, or use two sets of tab-set?

Reply:
Flattened the nesting and keeping one tab-set